L1Trigger/ME0Trigger/python/me0TriggerPseudoDigis_cff.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
+## configuration to build fast L1 ME0 trigger stubs
+## pseudo pads are created from pseudo digis with 192 strips instead of 384
+## the rechits are a necessary intermediate step before the pseudo pads are used
+## as input to build pseudo stubs
+
+from SimMuon.GEMDigitizer.muonME0PseudoReDigis_cfi import *
+from RecoLocalMuon.GEMRecHit.me0RecHits_cfi import *
+from RecoLocalMuon.GEMSegment.me0Segments_cfi import *
+
+simMuonME0PseudoReDigis192 = simMuonME0PseudoReDigis.clone(
```

Comment:
The "192" naming scheme seems very contingent to me. I would call it something more general like "Coarse" or "Half" that indicates it is less granular without using a specific number.